language-design.md

```diff
@@ -101,8 +201,30 @@ you choose to include natural language in the design of a DSL? If so, how might
 you do so? If not, why not?
 
 **Response**
-
-
+We believe that there should be a balance in designing programming languages
+between natural language and structured technical terms.
+
+The reason we cannot go to a full natural language is because there are so many
+grammars to the same language. For instance, in English, one stereotype is that
+a person from the South says "Y'all" while a person from the North says "You guys."
+Both words mean the same thing, and there are millions of variations of this,
+especially for a language as widespread as English.
+Additionally, one word can mean different things in different contexts. For
+example, in England, a rubber is an eraser while, in American slang, a rubber
+is a condom. This leads to confusion between humans who speak the same language,
+so it is extremely difficult to expect an artificial machine to understand the nuances.
```

Comment:
This seems to mean not that we can't use natural-language (or a language heavily based on natural language) but that we need to be very clear on the grammar and vocabulary restrictions of the language.